adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
We can take care of configuring the adapter with it's configuration and creating it atomicalling. However, each adapter will also need to be called with ConfigureMetric, Configure*Template during the config path.. Is configuring the `Type` considered as mutating a adapter ?